packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -36,200 +50,172 @@ class FramesBarPlotly {
-      ),
+      hoverlabel: HoverLabel(
+          font: Font(color: 'white'), bordercolor: 'rgba(255, 0, 0, .8)'),
+      marker:
+          Marker(color: cpuColor, line: Line(color: GPU_JANK_COLOR, width: 2)),
       width: [0],
     );
 
-/*
-    Data high_water_mark_cpu = Data(
-      type: 'scatter',
-      x: [],
-      y: [],
-      hoverinfo: 'none',
-      name: 'CPU Jank',
-      showlegend: false,
-      line: Line(
-          color: CPU_COLOR,
-          width: 1,
-          dash: 'dash'),
-    );
+    final Data traceGpuJank = Data(
+        y: [],
+        x: [],
+        type: 'bar',
+        name: 'GPU Jank',
+        hoverinfo: 'y+name',
+        hoverlabel: HoverLabel(
+            font: Font(color: 'white'), bordercolor: 'rgba(255, 0, 0, .7)'),
+        marker: Marker(
+            color: gpuColor, line: Line(color: GPU_JANK_COLOR, width: 2)),
+        width: [0]);
 
-    Data high_water_mark_gpu = Data(
-      type: 'scatter',
-      x: [],
-      y: [],
-      hoverinfo: 'none',
-      name: 'GPU Jank',
-      showlegend: false,
-      line: Line(
-          color: GPU_COLOR,
-          width: 1,
-          dash: 'dash'),
-    );
-*/
     // Return in trace index order
-    return [
-      trace_cpu_good,
-      trace_cpu_jank,
-      trace_gpu_good,
-      trace_gpu_jank,
-//      high_water_mark_cpu,
-//      high_water_mark_gpu
-    ];
+    return [traceCpuGood, traceCpuJank, traceGpuGood, traceGpuJank];
   }
 
-
   void plotFPS() {
-    Plotly.newPlot(_domName, createFPSTraces(),
-        getFPSTimeseriesLayout('Flutter FPS Rate'), Configuration(responsive: true,
-            displaylogo: false,
-            modeBarButtonsToRemove: ['sendDataToCloud', 'select2d', 'lasso2d',
-            'hoverClosestCartesian', 'hoverCompareCartesian', 'toggleSpikelines']));
+    Plotly.newPlot(
+        _domName,
+        createFPSTraces(),
+        getFPSTimeseriesLayout('Frame Rendering Time'),
+        Configuration(
+            responsive: true, displaylogo: false, displayModeBar: false));
   }
 
-  void plotFPSDataum(int dataIndex, num cpuDuration, num gpuDuration, bool paused) {
-    List<int> traces = [];
-
-    traces.add(cpuDuration > 8 ? CPU_JANK_TRACE : CPU_GOOD_TRACE);
-    traces.add(gpuDuration > 8 ? GPU_JANK_TRACE : GPU_GOOD_TRACE);
-//    traces.add(CPU_HIGH_WATER_MARK_TRACE);
-//    traces.add(GPU_HIGH_WATER_MARK_TRACE);
-
-    TraceData data = TraceData(
-      x: [[dataIndex], [dataIndex]],
-      y: [[cpuDuration], [gpuDuration]],
-//      x: [[dataIndex], [dataIndex], [dataIndex], [dataIndex]],
-//      y: [[cpuDuration], [gpuDuration], [8], [8]],
+  void plotFPSDatum(
+      int dataIndex, num cpuDuration, num gpuDuration, bool paused) {
+    final List<int> traces = [];
+
+    traces.add(cpuDuration > MS_JANK ? CPU_JANK_TRACE : CPU_GOOD_TRACE);
+    traces.add(gpuDuration > MS_JANK ? GPU_JANK_TRACE : GPU_GOOD_TRACE);
+
+    final TraceData data = TraceData(
+      x: [
+        [dataIndex],
+        [dataIndex]
```

Comment:
trailing comma here and after `[gpuDuration]` below

Reply:
Done